### PR TITLE
Mailerlite

### PR DESCRIPTION
**Pull request by maxharrisnet**
- Adding the Mailerlite global code snippet on all pages
- Swapping the form in the footer on all pages
- Override MailerLite's super specific styles because they only want you to use their GUI.

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -1,266 +1,252 @@
 <!DOCTYPE html>
```
Comment (maxharrisnet) [inline 404.html:1]: @SolivanLau hide whitespace

File: 404.html [hunk 2]
```diff
@@ -1,266 +1,252 @@
...
+						<li><a href="/impact.html">Our Impact</a></li>
+						<li><a href="/values.html">Our Values</a></li>
+						<li><a href="/events.html">Events & Programs</a></li>
+						<li><a href="/contact.html">Contact</a></li>
+						<li><a href="#">Support Us</a></li>
+					</ul>
+				</nav>
+
+				<!-- CAT Newsletter -->
+				<div class="ml-embedded" data-form="w80VC7"></div>
```
Comment (maxharrisnet) [inline 404.html:130]: Replacing the form with Mailerlite target div

File: 404.html [hunk 3]
```diff
@@ -1,266 +1,252 @@
...
+	<!-- google fonts -->
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link
+		href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap"
+		rel="stylesheet" />
+	<!-- styles -->
+	<link rel="stylesheet" href="css/global.css" />
+	<link rel="stylesheet" href="css/404.css" />
+	<!-- MailerLite Universal -->
```
Comment (maxharrisnet) [inline 404.html:21]: Adding MailerLite script to all pages

File: css/global.css [hunk 4]
```diff
@@ -888,23 +888,96 @@ footer .social {
...
-	color: var(--color-white);
-	display: flex;
-	flex-direction: column;
-	gap: var(--spacing-sm);
-	width: min-content;
+footer .social svg {
+	fill: var(--color-white);
+}
+
+.ml-embedded,
```
Comment (maxharrisnet) [inline css/global.css:896]: It's so ugly, don't look @SolivanLau !

File: 404.html [hunk 5]
```diff
@@ -1,266 +1,252 @@
...
+	<script>
```
Comment (SolivanLau) [inline 404.html]: looks good - can you please add this to **main.js** or another **component.js** file? I think it is easier to maintain this way.

**Review by SolivanLau — CHANGES_REQUESTED** [review]

**Comment by maxharrisnet** [comment]
@SolivanLau  I just made a few changes:
- Replaced code block with a script file
- Added form to policy pages (I think I forgot to merge them in before doing a find/replace)
- Styling the checkmark to be large

**Review by SolivanLau — APPROVED** [review]

**Pull request merged**